Maui.FreakyEffects/Samples/MauiProgram.cs

```diff
@@ -19,7 +18,8 @@ public static MauiApp CreateMauiApp()
             })
             .ConfigureEffects(effects =>
             {
-                effects.InitFreakyEffects();
```

Comment:
This should not fail?

Reply:
@Jon2G You can take a pull on master now, I have added the fix for this